app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -148,14 +149,13 @@ public String toString() {
          */
         String FULL_SYNC_END_MILLIS = "full_sync_end_millis";
 
-        /**
-         * The "snapshot time" of the last observation sync operation, according
-         * to the server's clock.  This is used to request an incremental update
-         * of observations from the server.
-         * <p/>
-         * <p>Updated after observation sync to the snapshot time reported by the server.
-         */
-        String OBS_SYNC_END_MILLIS = "obs_sync_end_millis";
+    }
+
+    public interface SyncTokens {
+        Uri CONTENT_URI = buildContentUri("sync-tokens");
+        String ITEM_CONTENT_TYPE = buildItemType("sync-token");
+        String TYPE = "type";
```

Comment:
Good call. It's much better with `table_name` instead of type, and I like the additional type-safety in the `SyncAdapter#[store/get]SyncToken` methods.